### ISSUE TITLE
`no-restricted-invocations` doesn't support modifiers

### ISSUE THREAD

**Issue by achambers** [ember-template-lint/ember-template-lint#2656]
I noticed today that the `no-restricted-invocations` rule doesn't support modifiers and it feels like it should.

Seems like it'd just take adding `ElementModifierStatement` to https://github.com/ember-template-lint/ember-template-lint/blob/9e998527cd0e495e750650f2fc73a24fd0c29fa7/lib/rules/no-restricted-invocations.js#L78-L83

Let me know if this is an improvement to the rule that you support and I'll knock together a PR.

**Comment by bmish** [comment]
Seems reasonable. As is, it's a breaking change that would need to go in with #2319.

**Comment by achambers** [comment]
> Seems reasonable. As is, it's a breaking change that would need to go in with #2319.

Coolio. Anything you need from me to make this so?

**Comment by bmish** [comment]
If you wanted to get it in sooner and also reduce risk, you could put this functionality behind an option (e.g. `includeModifiers` or something) and then we could just enable the option during the major release. But since it's not that big, it could also just go in during the major release as-is (whenever that happens).

**Comment by rwjblue** [comment]
I totally agree that its breaking, but I also think that we could consider it a bug fix. In other words, it seems pretty clear that this is a bug (we probably should have always considered modifiers here, but when the rule was written modifiers didn't exist), and semver clearly allows for the conceptual space of "breaking bugfixes".
